tests/test_git_colocated.rs

```diff
@@ -330,24 +330,24 @@ fn test_git_colocated_squash_undo() {
     test_env.jj_cmd_success(&repo_path, &["ci", "-m=A"]);
     // Test the setup
     insta::assert_snapshot!(get_log_output_divergence(&test_env, &repo_path), @r###"
-    @  rlvkpnrzqnoo 8f71e3b6a3be
-    ◉  qpvuntsmwlqt a86754f975f9 A master HEAD@git
+    @  rlvkpnrzqnoo 332adc48c3b9
+    ◉  qpvuntsmwlqt 9f48645b29b9 A master HEAD@git
     ◉  zzzzzzzzzzzz 000000000000
     "###);
 
     test_env.jj_cmd_success(&repo_path, &["squash"]);
     insta::assert_snapshot!(get_log_output_divergence(&test_env, &repo_path), @r###"
-    @  zsuskulnrvyr f0c12b0396d9
-    ◉  qpvuntsmwlqt 2f376ea1478c A master HEAD@git
+    @  zsuskulnrvyr 08a075b44988
+    ◉  qpvuntsmwlqt 5114fe510517 A master HEAD@git
     ◉  zzzzzzzzzzzz 000000000000
     "###);
     test_env.jj_cmd_success(&repo_path, &["undo"]);
     // TODO: There should be no divergence here; 2f376ea1478c should be hidden
```

Comment:
This comment also needs to be updated

Reply:
Fixed. I also searched for similar cases in comments with `// .*[0-9a-f]{4}` and didn't find any other cases. I found two other fixes to tests cases, which I added as new commits. PTAL.